dbt_checkpoint/check_model_has_tests_by_name.py

```diff
@@ -1,21 +1,23 @@
 import argparse
 import os
+import re
 import time
 from itertools import groupby
-from typing import Any, Dict, Optional, Sequence
```

Comment:
@Spince @SantiASC first of all, I see many unnecessary/unintended ungrouping of imports, this may be due to your Python formatter (I suggest you use Black), here and down in `utils`